src/schema/__tests__/order.test.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable promise/always-return */
+
+import { runQuery } from "test/utils"
+
+const exampleOrder = {
+  id: "fooid123",
+  telephone: "6073499419",
+  email: "",
+  line_items: [
+    {
+      quantity: 1,
+      artwork: {
+        id: "hubert-farnsworth-good-news",
+        title: "Good News",
+        artist: {
+          name: "Hubert Farnsworth",
+        },
+      },
+      partner: null,
+      partner_location: null,
+      shipping_note: null,
+      sale_conditions_url: null,
+    },
+  ],
+  shipping_address: {
+    name: "sarah sarah",
+    street: "401 Broadway, 25th Floor",
+    city: "New York",
+    region: "NY",
+    postal_code: null,
+    country: null,
+  },
+}
+
+describe("Order type", () => {
+  it("fetches order by id", () => {
+    const query = `
+      {
+        order(id: "52dd3c2e4b8480091700027f") {
+              id
+              telephone
+              email
+              line_items {
+                quantity
+                artwork {
+                  id
+                  title
+                  artist(shallow: true) {
+                    name
+                  }
+                }
+                partner {
+                  id
+                }
+                partner_location {
+                  id
+                }
+                shipping_note
+                sale_conditions_url
+              }
+              shipping_address {
+                name
+                street
+                city
+                region
+                postal_code
+                country
+              }
+            }
+      }
+    `
+
+    const rootValue = {
+      orderLoader: sinon.stub().returns(
```

Comment:
Don't need `sinon.stub()` here, can use `jest.fn().mockReturnValueOnce(...)`.

That also allows you make an assertion on the argument that `orderLoader` took (namely, `{id: "...."}`).

Reply:
👍